src/styles/App.css

```diff
@@ -0,0 +1,3 @@
+#app-wrapper-div {
```

Comment:
This should use flex box ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)
Also the display and buttonpanel components, please make changes accordingly

Reply:
Ok will do that